cmd/agent/api/agent/agent.go

```diff
@@ -219,6 +220,25 @@ func getFormattedStatus(w http.ResponseWriter, r *http.Request) {
 	w.Write(s)
 }
 
+func getCheckMemoryStatus(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	component := vars["component"]
+	switch component {
+	case "py":
+		jsonStatus, err := status.GetPythonCheckMemoryUsage(common.Coll)
+		if err != nil {
+			log.Errorf("unable to collect python check memory usage: %s", err.Error())
+			http.Error(w, err.Error(), 500)
+		}
```

Comment:
Do we need an `else` here?

Reply:
No, but what I am missing is a return after writing the `500`. Let me clean that up! Thanks for the catch.